### DISCUSSION TITLE
Add support for Cursor as an IDE

### DISCUSSION THREAD

**Discussion by jlpellicer** [beyondcode/herd-community#413] [Ideas]
Cursor is an AI-powered code editor forked from VS Code.
Not affiliated, just a fan. I would like to see it added as an IDE.
[Cursor](https://cursor.sh)

**Comment by joshuaoliver** [comment]
Agree with this 100%

**Comment by Borcioo** [comment]
up?

**Comment by dr-codswallop** [comment]
Agreed, and possibly also [Windsurf](https://codeium.com/windsurf) & [Trae](https://www.trae.ai) … all forks of VSCode which are gaining popularity. 

Maybe we could have a config file where we can specify our own IDE's, similar to [Browsersync options](https://browsersync.io/docs/options#option-browser).
Or a UI like Herd Paths allowing us to specify our IDE's.
